File: 30 days of code/__ Java/Day 00 - Hello world.java
/**
 *              HELLO, WORLD | JAVA
 * Domain       30 DAYS OF CODE
 * Sub domain   DAY 00
*/

import java.io.File;
import java.io.FileNotFoundException;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) throws FileNotFoundException {
        File file = new File("input.txt");
        Scanner scan = new Scanner(file);
        String s = scan.nextLine();

        System.out.println("Hello, World.");
        System.out.println(s);

        scan.close();
    }
}

File: 30 days of code/# C#/Day 00 - Hello world.cs
/**
 *              HELLO, WORLD | C#
 * Domain       30 DAYS OF CODE
 * Sub domain   DAY 00
*/

using System;
using System.IO;

class Program
{
    //static TextReader read = Console.In;
    static StreamReader read = new StreamReader("input.txt");

    static void Main(string[] args)
    {
        string s = read.ReadLine();
        read.Close();

        Console.WriteLine("Hello, World.");
        Console.WriteLine(s);
    }
}